Escape @ characters in ASCII art diagrams in internals.texi.
Modern texinfo treats @| as a command even inside @example blocks.
PLD-specific fix for texinfo 7.x compatibility.

--- xemacs-21.5.34/man/internals/internals.texi.orig	2013-08-21 19:43:45.000000000 +0200
+++ xemacs-21.5.34/man/internals/internals.texi	2026-03-15 12:00:00.000000000 +0100
@@ -18749,7 +18749,7 @@
 | #  | +----------------------------------------------------------+ |  # |
 | #  | |                          gutter                          | |  # |
 | #  | |-********************************************************-| |  # |
-|w#  | | *@|        scrollbar        |v*                      |s* | |  #w|
+|w#  | | *@@|        scrollbar        |v*                      |s* | |  #w|
 |i#  | | *-+-------------------------|e*                      |c* | |  #i|
 |n#  | | *s|                         |r*                      |r* | |  #n|
 |d#  | | *c|                         |t*                      |o* | |  #d|
@@ -18759,7 +18759,7 @@
 |m#  |t| *l|                         |i*                      |a* |t|  #m|
 |a#  |e| *b|                         |v*                      |r* |e|  #a|
 |n# t|r| *a|                         |i*----------------------+-* |r|t #n|
-|a# o|n|g*r|                         |d*      scrollbar       |@*g|n|o #a|
+|a# o|n|g*r|                         |d*      scrollbar       |@@*g|n|o #a|
 |g# o|a|u*-+-------------------------|e*----------------------+-*u|a|o #g|
 |e# l|l|t*        modeline           |r*      modeline          *t|l|l #e|
 |r# b| |t********************************************************t| |b #r|
@@ -18773,13 +18773,13 @@
 |t#  | | *e  = inner text area =e  |a|i*                      |a* | |  #t|
 |i#  | | *   =                 =   |r|v*                      |r* | |  #i|
 |o#  | | *---===================---+-|i*----------------------+-* | |  #o|
-|n#  | | *        scrollbar        |@|d*      scrollbar       |@* | |  #n|
+|n#  | | *        scrollbar        |@@|d*      scrollbar       |@@* | |  #n|
 | #  | | *-------------------------+-|e*----------------------+-* | |  # |
 | #  | | *        modeline           |r*      modeline          * | |  # |
 | #  | |-********************************************************-| |  # |
 | #  | |                           gutter                         | |  # |
 | #  | |-********************************************************-| |  # |
-| #  | |@*                       minibuffer                     *@| |  # |
+| #  | |@@*                       minibuffer                     *@@| |  # |
 | #  | +-********************************************************-+ |  # |
 | #  |                         internal border                      |  # |
 | #--------------------------------------------------------------------# |
